samples/snippets/storage_async_upload.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+
+# Copyright 2021 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import asyncio
+import sys
+
+
+"""Sample that asynchronously uploads a file to GCS
+"""
+
+
+# [START storage_async_upload]
+async def async_upload_blob(bucket_name):
```

Comment:
Can you add a comment that the sample can be run, in practice, with asyncio.run() as it is in the tests? So that people unfamiliar with async functions know they don't have to change their whole program.